draft-irtf-cfrg-vdaf.md

```diff
@@ -2206,6 +2206,15 @@ methods refer to constants enumerated in {{prio3-const}}.
 | `USAGE_JOINT_RAND_PART: Unsigned`  | 7     |
 {: #prio3-const title="Constants used by Prio3."}
 
+A list of Prio3 specific parameters, in {{prio3-specific-params}}, are provided by each instantiation {{prio3-instantiations}}.
```

Comment:
In POC, `PROOFS` lives in `class Prio3`. So I didn't add this parameter in VDAF or FLP. What do you think?

Reply:
I agree I don't think it belongs in either table.

I would probably not create a table just for this. The first paragraph of {{prio3-construction}} (i.e., this section) lists `Xof` and `Flp` as "associated parameters"; I'd define `PROOFS` there. Perhaps call it an "associated constant".